docs/conf.py

```diff
@@ -6,18 +6,22 @@
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-import nemos
 import sys, os
 from pathlib import Path
 
+from importlib.metadata import version
```

Comment:
I think this bit only shows up here because I messed up: it's from https://github.com/flatironinstitute/nemos/pull/308, which was merged into `main`, not `development`. We should merge `main` back into `development`, so this goes away.